Wikipedia/Code/WMFArticleImageProtocol.m

```diff
@@ -23,6 +23,22 @@
 
 NSString* const WMFArticleImageSectionImageRetrievedNotification = @"WMFSectionImageRetrieved";
 static NSString* const WMFArticleImageProtocolHost               = @"upload.wikimedia.org";
+static NSString* const WMFArticleImageProtocolResponseAndDataErrorDomain = @"WMFArticleImageProtocolResponseAndDataErrorDomain";
```

Comment:
`WMFArticleImageProtocolErrorDomain` sound be enough - domains can usually just be scoped to the class